src/picturepark-sdk-v1-angular/src/app/components/application-menu/application-menu.module.ts

```diff
@@ -2,10 +2,10 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { RouterModule } from '@angular/router';
 
-import { MaterialsModule } from '../../materials-module/materials-module.module';
 
 // COMPONENTS
 import { ApplicationMenuComponent } from './application-menu.component';
+import { MaterialsModule } from 'projects/picturepark-sdk-v1-angular-ui/src/lib/materials-module/materials-module.module';
```

Comment:
This should be imported from @picturepark/sdk-v1-angular-ui

Reply:
On a second note, I don't think it is a good practice to import this module from _picturepark-sdk-v1-angular-ui_.
_picturepark-sdk-v1-angular-ui_ already references _picturepark-sdk-v1-angular_, so we end up having circular references.
In my opinion, the best way is to move the Materials module to the _picturepark-sdk-v1-angular_ project or, if not possible, copy it to _picturepark-sdk-v1-angular_.